lib/debug/session.rb

```diff
@@ -239,6 +239,7 @@ def wait_command
       #   * Finish debugger (with the debuggee process on non-remote debugging).
       when 'q', 'quit'
         if ask 'Really quit?'
+          disable_tracer
```

Comment:
why does it needed?

Reply:
because debugger would hang when tracer is on. you can reproduce the bug with

```
$ exe/rdbg -e  'trace on;; quit!' target.rb
```

Reply:
Thank you. I'll check it.

Reply:
9d8f49389c24680ea53370c2d9210d1485b9ee5c may fix this issue.

Reply:
for checking `at_exit` or similar feature, trace will be useful.

Reply:
@ko1 cool thanks. I'll drop that commit.